sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+#### Differences in the sBTC Bootstrap Phase:
+- sBTC Signer eligibility criteria will be confirmed through community governance vote. The eligibility criteria to become an sBTC signer is described below and will take into account Signer performance and availability. sBTC Signers that meet the criteria are confirmed by the sBTC Working Group. A subsequent release will implement an open and dynamically rotating signer set for each Proof of Transfer cycle, as described in the original sBTC design documents.
+- sBTC Bootstrap Signers are separate from Stacks Signers. In a subsequent sBTC release, and accompanying Stacks network hard fork, Stacks signers will be required to become sBTC signers. The bootstrapping period uses a separate subset of Stacks signers to secure the sBTC protocol and Signer operations are not explicitly linked with slashing of rewards.
+- sBTC deposits will be triggered via an API call. During the bootstrap phase sBTC deposits must be initiated via an API call to alert the signers to the presence of a deposit. After the bootstrapping phase users will have the option to alert the Signers of a deposit via a Stacks contract call, but the API will remain as a fee-less option to communicate with the Signers.
+
+These changes enable sBTC to be launched earlier and with less risk than the long term decentralized approach. The sBTC release with an open and dynamic signer set is expected to be released early 2025, which would enshrine the sBTC protocol into Stacks consensus.
+
+## Specification
+
+### Deposits (Minting sBTC)
+An API call referencing a Bitcoin transaction initiates the deposit process that can be completed within a single Bitcoin block. The API is separate from the sBTC Bootstrap Signers but will need access to the sBTC Bootstrap Signers to let them know which transactions to pick up.
+
+**API:** The API relays deposit information to the sBTC Bootstrap Signers. The API reduces the cost for deposit on rejection, reduces the scope of sBTC, and makes the sBTC system easier to service.
+
+**Bitcoin Deposit Information:** The deposit will need to pay to taproot and be spendable by a consensus threshold of the signers. The API will need to know all of the taproot spend paths and the spend paths will need to fit a specific format that cannot be clawed back in the short term in order for the Signers to consider approving it.
+
+#### Deposit Flow
+The main steps of the sBTC Deposit flow are as follows:
+1. **Deposit request:** A bitcoin holder creates a transaction on Bitcoin.
+    - The deposit transaction contains a UTXO (deposit UTXO) spendable by sBTC Signers, with an OP_DROP payload.
```

Comment:
The wire format for this needs to be described, in byte-by-byte detail.

Reply:
Described here, we can add some of these details: https://github.com/stacks-network/sbtc/issues/30

Below is a snippet:

> ### Requirements of the spend conditions
> The spend conditions are BIP-342 compliant tapscripts, with additional requirements on their content. The accept condition $s_a$ must be of the form
> 
> ```
> <sbtc_payload> DROP <signers_pubkey> CHECKSIGVERIFY
> ```
> 
> where the `sbtc_payload` contains the following data
> 
> ```
> 0               8                                          159
> |---------------|-------------------------------------------|
>    max_fee             recipient_address
> ```
> 
> where `max_fee` is the maximum fee the BTC holder would like to pay for the deposit, and the `recipient_address` is the address which should be minted the sBTC, encoded as a Clarity value as defined in [SIP-005](https://github.com/stacksgov/sips/blob/main/sips/sip-005/sip-005-blocks-and-transactions.md#clarity-value-representation).
> 
> The reclaim condition can be any spend condition as long as it is initiated with a timelock
> 
> ```
> <lock_time> CHECKSEQUENCEVERIFY <reclaim script>
> ```
> 
> The timelock allow Signers to choose to reject transactions if there is a risk that a user may attempt reclaim the deposit UTXO in the same block as the sBTC transaction which would consume the same UTXO, which would increase the risk of the sBTC transaction not being included in a block.

Reply:
Great; please add it to the SIP.